src/server.rs

```diff
@@ -64,7 +64,8 @@ pub struct Server<F> {
     is_first_server: bool,
     accumulator: Vec<F>,
     validation_mem: ValidationMemory<F>,
-    private_key: PrivateKey,
+    /// The private key used by the server to decrypt inputs
```

Comment:
Can the code be written without making this public? As a general rule it's a good idea to keep secret values private in order to avoid API misuse.

Reply:
Good call. This is needed so that `tests/tweaks.rs` can use it, and integration tests have to use public API. But clearly `tweaks.rs` wants to poke at library internals, so I think it'd be appropriate to make it a unit test and not expose this externally.